Podcast/EpisodeTableViewCell.swift

```diff
@@ -0,0 +1,357 @@
+//
```

Comment:
We should keep it separate. The feed cells might significantly change since they are initialized from cards and not episodes.